proc/proc.go

```diff
@@ -91,18 +90,8 @@ func CompileProc(custom Compiler, node ast.Proc, c *Context, parent Proc) ([]Pro
 	}
 	switch v := node.(type) {
 	case *ast.ReduceProc:
-		reducers := make([]compile.CompiledReducer, 0)
-		for _, reducer := range v.Reducers {
-			compiled, err := compile.Compile(reducer)
-			if err != nil {
-				return nil, err
-			}
-			reducers = append(reducers, compiled)
-		}
-		params := ReduceParams{
-			reducers: reducers,
-		}
-		return []Proc{NewReduce(c, parent, params)}, nil
+		g := &ast.GroupByProc{Reducers: v.Reducers}
```

Comment:
(The endgame is probably to change zql.peg to just emit an ast.GroupByProc and remove ast.ReduceProc altogether; I'd rather do that in a separate step right after wrapping up #1004 )